student.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, name = 'Unknown')
+    super(false)
+    @name = name
+    @classroom = classroom
+  end
```

Comment:
- [ ] Student class should inherit name, parent permission and age from the Person class, so it should be: 

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```